src/docs/release-notes/R25/FullNotes.mdx

```diff
@@ -0,0 +1,38 @@
+### Angular and React
+
+-   Migrated our component library documentation site from Storybook to an in-house solution. We have added some powerful new features to our documentation, such as Examples pages that provide accurate code snippets and context for each component we support. We also introduced Playground pages, which generates a working, copy/pastable snippet based on user input.
+    -   [Angular Developer Docs](https://brightlayer-ui-components.github.io/angular)
+    -   [React Developer Docs](https://brightlayer-ui-components.github.io/react)
+-   Increased code test coverage for authentication workflows and design patterns.
+
+### React Native
+
+-   Added support for React Native v0.70.
```

Comment:
We didn't exactly _add_ support. More like 'Updated our packages to work with React Native v0.70.x'